tests/neg-custom-args/captures/capture-vars-subtyping.scala

```diff
@@ -0,0 +1,47 @@
+import language.experimental.captureChecking
+import caps.*
+
+def test[C^] = 
+  val a: C = ???
+  val b: CapSet^{C^} = a
+  val c: C = b
+  val d: CapSet^{C^, c} = a
+
+// TODO: make "CapSet-ness" of type variables somehow contagious?
+// Then we don't have to spell out the bounds explicitly...
```

Comment:
We should support that. Say we have `def foo[C^, D <: C, E >: C]`, then we should automatically constraint both `D` and `E` to be capture set variables, i.e., `D` is implicitly lower-bounded by `CapSet`, and `E` is implicitly upper-bounded by `CapSet^`